### ISSUE TITLE
[Explore]: CF CLI user can stream some data via `cf logs`

### ISSUE THREAD

**Issue by acosta11** [cloudfoundry/korifi#1295]
### Background

The CF CLI logs command makes a `api/v1/read` request with a persistent connection to stream log data. We would like to understand the shape of an api handler response to satisfy the api at which point the mapping from the kubernetes `client-go` log tail will become clear.

The CF CLI relies on the go-log-cache library for its requests to log cache, in particular the walk function: https://github.com/cloudfoundry/go-log-cache/blob/main/walk.go .

While the Reader interface in the walk function indicates a request need only produce []*loggregator_v2.Envelopes to subsequently pass into a Visitor for consumption, there is not a clear requirement for the persistence of the connection used by the client? Based on an initial look at our configuration of the client in [the cli](https://github.com/cloudfoundry/cli/blob/d3bb1ecfc94ee12432f8defb30ab9459e83cc727/api/logcache/log_cache_client.go#L127), we appear to default to an http client making singular read requests as opposed to persistent grpc connection. So getting streaming working may just amount to debugging errors in the actual read requests.

Note, exploring this with the cli may depend on #1294 or a local hack to the cli to remove the token refresh call, as there may be noisy errors with the `oauth/token` request?

### Acceptance Criteria

**GIVEN** a CF CLI w/ version 8.4.0 or later
**AND GIVEN** a standard installation of korifi
**When** I push an app and run `cf logs`
**THEN** the log streaming request of the form `GET api/v1/read` successfully returns some data

Follow-up issue for the actual api implementation of the /api/v1/read endpoint.

### Timebox

No timebox

**Comment by acosta11** [comment]
We ended up making a dev build of the cli to bypass the oauth issue and discovered that runtime log streaming was already working. For staging logs on cf push, we identified a missing "source_type" tag on the log envelopes causing them to be filtered out by the cli.

Follow up issues created to finish implementing streaming logs:
* #1294 
* #1298
